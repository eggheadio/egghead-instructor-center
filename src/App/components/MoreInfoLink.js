import React from 'react'

import Link from './Link'

const MoreInfoLink = ({url}) => (
  <Link url={url}>
    <span className='fa fa-info-circle' />
  </Link>
)

MoreInfoLink.propTypes = {
  url: React.PropTypes.string.isRequired,
}

export default MoreInfoLink

