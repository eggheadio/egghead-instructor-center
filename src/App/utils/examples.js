export default {
  text: {
    short: 'Lorem ipsum',
    medium: 'Lorem ipsum dolor sit amet, urna erat feugiat egestas, metus molestie, adipiscing sed nisl et, dictumst erat sem erat.',
  },
  url: 'https://egghead.io',
  markdown: 'Lorem ipsum *dolor*',
}
