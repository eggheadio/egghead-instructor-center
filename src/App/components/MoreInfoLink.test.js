import React from 'react'
import renderer from 'react-test-renderer'

import examples from '../../App/utils/examples'

import MoreInfoLink from './MoreInfoLink'

test('Default', () => {
  const tree = renderer.create(<MoreInfoLink 
    url={examples.url}
  />)
  expect(tree).toMatchSnapshot()
})
