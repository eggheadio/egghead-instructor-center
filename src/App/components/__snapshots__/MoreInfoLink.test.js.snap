exports[`test Default 1`] = `
<a
  className="blue"
  href="https://egghead.io"
  target="_blank">
  <span
    className="fa fa-info-circle" />
</a>
`;
